ansible/roles/common_installs/tasks/main.yml

```diff
@@ -92,7 +93,6 @@
   pip: name={{ item }}
   sudo: yes
   with_items:
-    - fabric
```

Comment:
@dannyroberts do you know why this is here? i don't think this is used anywhere so figured i'd remove